app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -1130,6 +1130,74 @@ def test_serializer_feature_config_validation_treatment_value_no_schema(self):
         )
         self.assertTrue(serializer.is_valid())
 
+    def test_rollout_valid_version_support(self):
```

Comment:
So you could parameterize these tests to test every application/version pair like we do [here](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py#L239-L259) for both the valid and invalid cases, that way they're all covered.

Reply:
I wanted to ask you about that. I have a few questions (e.g. do we want to include klar in this list) so I think I'd like to land this in the meantime and chat about it the version #s

Reply:
Ya totes, not a blocking thing, just worth pointing out.

Reply:
https://github.com/mozilla/experimenter/issues/7584